extensions/authentication-passport/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@loopback/authentication-passport",
+  "version": "0.1.0",
```

Comment:
Let's start with `0.0.1` so that `0.1.0` will be the 1st published version.

Reply:
@raymondfeng we already have `@loopback/authentication-passport@0.1.0` published, see https://www.npmjs.com/package/@loopback/authentication-passport.

So this time it should be published as 0.2.0(due to the upgrade of `@loopback/authentication@3.x`)

Reply:
I see :-).